src/dataflow-types/src/types.rs

```diff
@@ -1183,6 +1183,14 @@ pub mod sources {
         !is_avro && !is_stateless_dbz
     }
 
+    #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+    pub enum ConnectorLiteral {
```

Comment:
`ConnectorLiteral` feels like an odd name to me--maybe `ConnectorInner` or `ConnectorValues`? Not a huge deal, but a `ConnectorLiteral` feels like it also implies a `ConnectorReference`.

Reply:
This name was originally because it was the dataflow-types counterpart to the `KafkaConnector::Literal` which has since become `KafkaConnector::Inline` but that name doesn't feel appropriate here either. I think your suggestion of `ConnectorInner` seems good though.

Reply:
Should this not just be `Connector`?

Reply:
This creates an immense number of naming conflicts which rapidly got overwhelming to try and unravel because it clashes with the AST type which contains this dataflow type as a field.

Reply:
Ah, ok, I see.